contrib/depends/packages/unbound.mk

```diff
@@ -17,7 +16,10 @@ define $(package)_set_vars
   $(package)_cflags_mingw32+="-D_WIN32_WINNT=0x600"
 endef
 
+# Remove blobs
 define $(package)_preprocess_cmds
+  rm configure~ doc/*.odp doc/*.pdf contrib/*.tar.gz contrib/*.tar.bz2 &&\
```

Comment:
Could make this a little easier to read with alternate shell syntax:
```rm configure~ doc/*.{odp,pdf} contrib/*.tar.{gz,bz2}  ...```

Reply:
Brace expansion isn't defined in POSIX. I didn't know this until I found out that:

In `depends` CI, which runs on Ubuntu 20.04, the `SHELL` environment variable isn't set by default causing `make` to fall back to `/bin/sh`, resulting in [an error](https://github.com/monero-project/monero/actions/runs/12895008127/job/35954977674?pr=9642#step:8:33) on this line. On 20.04 `/bin/sh` is [Dash](https://en.wikipedia.org/wiki/Almquist_shell#Dash) which tries to be POSIX compliant and does not support brace expansion.

For portability reasons, I have reverted back to the less readable syntax.